doc/api/crypto.md

```diff
@@ -1791,6 +1791,11 @@ but will take a longer amount of time to complete.
 The `salt` should be as unique as possible. It is recommended that a salt is
 random and at least 16 bytes long. See [NIST SP 800-132][] for details.
 
+The `digest` should either be a string value or null:
```

Comment:
I think we can just shorten this to just mention that if `digest` is `null`, then `'sha1'` is used, or perhaps include that explanation earlier in the function description where `digest` is first mentioned.

Reply:
OK, I'll take this advice to report the truth, but I'll also notice that this is deprecated. BTW：When https://github.com/nodejs/node/pull/22861 is merged, a warning will be also given.

Reply:
Take this adive! :)
On the one hand, it tells the truth, though we've missed that by accident.
On the other hand, this has given a good explaination to tell users that this value of null is deprecated since v11.

Reply:
What I was mainly getting at about this line is that the part about needing to be a string value is unnecessary as earlier in the description it's noted that `digest` should be an HMAC digest algorithm, which implicitly means a string value.